roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -32,6 +32,12 @@
   when: cloud_provider is defined and cloud_provider == 'azure'
   ignore_errors: "{{ ignore_assert_errors }}"
 
+- name: Stop if unsupported version of Kubernetes
+  assert:
+    that: kube_version is version(kube_version_min_required, '<')
+    msg: "The current release of Kubespray only support newer vesion of Kubernetes than: {{ kube_version_min_required }}"
```

Comment:
Should we include the the `kube_version` in that error message?

Reply:
Added :)